<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>noaa daily all browser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionopen"/>
   <addaction name="actionstop"/>
   <addaction name="actionfind"/>
   <addaction name="actionsort"/>
   <addaction name="actionclear_find"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockStations">
   <property name="windowTitle">
    <string>observations</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="query">
         <property name="toolTip">
          <string>query</string>
         </property>
         <property name="text">
          <string>element='TAVG' and range(year,2000,2017)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="limit">
         <property name="toolTip">
          <string>rows found limit (0=no limit)</string>
         </property>
         <property name="prefix">
          <string>limit </string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>500000</number>
         </property>
         <property name="singleStep">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>sort/group</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QComboBox" name="sortOrder">
          <item>
           <property name="text">
            <string>ascending</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>descending</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="sortFields">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="stGlobal">
          <property name="text">
           <string>results:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockGroup">
   <property name="windowTitle">
    <string>tree group</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="TreeGroup" name="tree">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="AuxFileCombo" name="cbElements">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>elements</string>
       </property>
       <property name="accessibleName">
        <string>elements</string>
       </property>
       <property name="styleSheet">
        <string notr="true">combobox-popup: 0;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AuxFileCombo" name="cbCountries">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>countries</string>
       </property>
       <property name="accessibleName">
        <string>countries</string>
       </property>
       <property name="styleSheet">
        <string notr="true">combobox-popup: 0;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AuxFileCombo" name="cbStates">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>states</string>
       </property>
       <property name="accessibleName">
        <string>states</string>
       </property>
       <property name="styleSheet">
        <string notr="true">combobox-popup: 0;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>stations</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="TableAuxFile" name="stations">
              <property name="font">
               <font>
                <family>Courier New</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="queryStations">
           <property name="toolTip">
            <string>any field in station contains (case insensitive)...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockGraph">
   <property name="windowTitle">
    <string>graph</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tabLine">
        <attribute name="title">
         <string>line</string>
        </attribute>
        <layout class="QVBoxLayout" name="layOutLine">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
        </layout>
       </widget>
       <widget class="QWidget" name="tabLineSeries">
        <attribute name="title">
         <string>Line series</string>
        </attribute>
        <layout class="QVBoxLayout" name="layOutLineSeries">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
        </layout>
       </widget>
       <widget class="QWidget" name="tabBar">
        <attribute name="title">
         <string>bar</string>
        </attribute>
        <layout class="QVBoxLayout" name="layOutBar">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionstop">
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="actionclear_find">
   <property name="text">
    <string>clear find</string>
   </property>
  </action>
  <action name="actionfind">
   <property name="text">
    <string>find</string>
   </property>
  </action>
  <action name="actionsort">
   <property name="text">
    <string>sort</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TreeGroup</class>
   <extends>QTreeView</extends>
   <header location="global">treegroup.h</header>
  </customwidget>
  <customwidget>
   <class>AuxFileCombo</class>
   <extends>QComboBox</extends>
   <header location="global">comboboxmodel.h</header>
  </customwidget>
  <customwidget>
   <class>TableAuxFile</class>
   <extends>QTableView</extends>
   <header location="global">tableauxfile.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
